src/test/java/uk/gov/pay/api/it/PaymentsResourceGetIT.java

```diff
@@ -765,6 +778,21 @@ public void getPaymentWithNoAuthorisationSummaryThroughConnector() {
         response.body("authorisation_summary", is(nullValue()));
     }
 
+    @Test
+    public void getPaymentWithNoAuthorisationSummaryThroughConnectorWhenRequiredThreeDSecureIsFalse() {
```

Comment:
This name is inconsistent with the others (in terms of whether or not there’s `No` and `Required`).